lib/render/mayaToHydra/utils.h

```diff
@@ -36,12 +37,22 @@ struct MtohRendererDescription {
 };
 
 using MtohRendererDescriptionVector = std::vector<MtohRendererDescription>;
+using MtohRendererSettingsVector = std::vector<HdRenderSettingDescriptorList>;
 
 TfTokenVector MtohGetRendererPlugins();
```

Comment:
With your changes, `MtohGetRendererPlugins` can now be removed entirely!  It's currenly only used in one place, - [`MtohViewCmd::doIt`, in `viewCommand.cpp`](https://github.com/Autodesk/maya-usd/blob/1b5f27a7e70053826bab23f0bd0678aaccb1b470/lib/render/mayaToHydra/viewCommand.cpp#L144) which can be easily adapted to simply iterate over `MtohGetRendererDescriptions` instead.

Reply:
Made a separate PR to remove this function entirely:

https://github.com/Autodesk/maya-usd/compare/dev...LumaPictures:pr/hdmaya/remove_MtohGetDefaultRenderer?expand=1

I'll get it merged once this one goes through...

Reply:
It looks like you removed the implementation and usage, but forgot to remove the declaration here...?